app/models/create_expenses_with_receipt.rb

```diff
@@ -0,0 +1,35 @@
+class CreateExpensesWithReceipt
+  attr_reader :expense
+
+  def initialize(expense_params)
+    @params = expense_params
+  end
+
+  def create
+    setup
+    unless @expense.valid?
+      merge_errors
+      return false
+    end
+    do_saves
```

Comment:
What about `save`? It would make extra sense because it returns a boolean, such as the AR `save`.